### PR TITLE
Release dynflow 0.8.5 (RPM)

### PR DESCRIPTION
**Pull request by stbenjam**

### CODE REVIEW

**Comment by iNecas** [comment]
ACK

**Comment by iNecas** [comment]
0.8.5 will be a bit better due to https://github.com/Dynflow/dynflow/pull/166 :)

File: rubygem-dynflow/rubygem-dynflow.spec [hunk 1]
```diff
@@ -27,7 +26,7 @@ Requires: %{?scl_prefix}rubygem(concurrent-ruby) < 0.10.0
 Requires: %{?scl_prefix}rubygem(concurrent-ruby-edge) >= 0.1.0
 Requires: %{?scl_prefix}rubygem(concurrent-ruby-edge) < 0.2.0
 Requires: %{?scl_prefix}rubygem(multi_json)
-Requires: %{?scl_prefix}rubygem(apipie-params)
+Requires: %{?scl_prefix}rubygem(apipie-params) >= 0.0.4
```
Comment (domcleal) [inline rubygem-dynflow/rubygem-dynflow.spec]: The req is on >= 0, so not sure why you added this?

**Comment by domcleal** [comment]
0.8.5's out now.

**Comment by stbenjam** [comment]
@domcleal Updated

**Comment by domcleal** [comment]
Thanks @stbenjam, building.

**Pull request merged**